samples/lib/thrift/hello_client/src/main.cpp

```diff
@@ -14,16 +14,22 @@
 #include <fcntl.h>
 
 #include <thrift/protocol/TBinaryProtocol.h>
+#include <thrift/protocol/TCompactProtocol.h>
 #include <thrift/transport/TBufferTransports.h>
-#include <thrift/transport/TFDTransport.h>
+#include <thrift/transport/TSSLSocket.h>
 #include <thrift/transport/TSocket.h>
+#include <thrift/transport/TZlibTransport.h>
 
 #include "Hello.h"
 
 using namespace apache::thrift;
 using namespace apache::thrift::protocol;
 using namespace apache::thrift::transport;
 
+#ifndef IS_ENABLED
```

Comment:
This likely won't work as expected outside of Zephyr

Reply:
I defined the predicates of `IS_ENABLED` to 0 or 1 in Makefile, so expanding the predicate as-is has the same effect in Zephyr

Reply:
Hmm.. yeah, that's kind of clever 🤔 Does the job, I guess.